directives/animate/animate.js

```diff
@@ -1,8 +1,8 @@
-import { directive, PropertyPart } from 'lit/directive.js';
+import { directive, PartType } from 'lit-html/directive.js';
```

Comment:
The first error I was getting as that `PropertyPart` wasn't importing at all. The class/type itself didn't end up being needed since the new `PartType` info object has that stuff on it anyway.

Reply:
Nice. Yeah I was stuck on this one for a while since it just didn't import. Lucky that we can swap in the PartType here.